src/test/java/org/example/homework15/DemoTestSecondPart.java

```diff
@@ -0,0 +1,102 @@
+package org.example.homework15;
+
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.openqa.selenium.support.ui.ExpectedConditions.alertIsPresent;
+
+public class DemoTestSecondPart {
+    WebDriver driver;
+    JavascriptExecutor js;
+    private static final By ALERTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Alerts, Frame & Windows']");
+    private static final By ALERTS_MENU = By.xpath("//div[@class='element-list collapse show']/ul/li[2]");
```

Comment:
`//div[@class='element-list collapse show']/ul/li[2]` you are trying to point the element by index,
I see the roots of that decision, but this is not the best option. In this case better to rely on text(since we have no localization here this option would be reasonable)